newton/_src/solvers/mujoco/solver_mujoco.py

```
@@ -1042,6 +1303,165 @@ def _init_tendons(self, model: Model, spec, joint_mapping: dict[int, str], templ
+            actuator_args: Default actuator arguments.
+            mjc_actuator_ctrl_source_list: List to append control sources to.
+            mjc_actuator_to_newton_idx_list: List to append Newton indices to.
+            dof_to_mjc_joint: Mapping from Newton DOF index to MuJoCo joint index.
+                Used to resolve CTRL_DIRECT joint actuators to their MuJoCo targets.
+            mjc_joint_names: List of MuJoCo joint names indexed by MuJoCo joint index.
+                Used together with dof_to_mjc_joint to get the correct joint name.
+
+        Returns:
+            int: Number of actuators added.
+        """
+        import mujoco  # noqa: PLC0415
```

Ruff (0.14.14)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)